### PR TITLE
[Feature #163460553] Consume GET all meetup API-endpoint

### PR DESCRIPTION
**Pull request by abidex4yemi**
### What does this PR do?

  Consume get all meetup API-endpoint and connect it to the front-end
  page

### Description of the task to be completed?

  * Consume and connect get all meetup API-endpoint with all meetup
  front-end page

  * Have the following task completed

    `Make a POST request to /meetup endpoint`

    ` Check for success status message`

    ` Get all returned data`

    ` Display returned data`

    ` If no record show no record available`

### How should this be manually tested?

  * By visiting the link listed below

    - All meetup page

      [https://abidex4yemi.github.io/meet-up-questioner/meetups.html](https://abidex4yemi.github.io/meet-up-questioner/meetups.html)

  * By clicking on a specific meetup to view details

### What are the relevant pivotal tracker stories?

Closes: #163460553

### CODE REVIEW

File: UI/js/all-meetup.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
+const feedbackContainer = document.querySelector('.feedback-message');
+
+function showOverlay() {
+  document.querySelector('.overlay').style.display = 'block';
```
Comment (andela-gike) [inline UI/js/all-meetup.js:4]: can you make all these function arrow function

File: UI/js/meetup-details.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
+/**
+ * get meetup id from url parameter
+ *
+ * @param {void}
+ *
+ * @returns {number} meetupId
+ */
+function getId() {
```
Comment (andela-gike) [inline UI/js/meetup-details.js]: Same here, make every function arrow functions

**Review by andela-gike — CHANGES_REQUESTED** [review]

File: UI/js/all-meetup.js (hunk 1 above)
Comment (abidex4yemi) [reply UI/js/all-meetup.js:4]: ok, i will convert it now.

Comment (abidex4yemi) [reply UI/js/all-meetup.js:4]: I have made the changes.

**Comment by abidex4yemi** [comment]
Done.

File: UI/js/meetup-details.js (hunk 2 above)
Comment (abidex4yemi) [reply UI/js/meetup-details.js]: Done.

File: UI/js/all-meetup.js (hunk 1 above)
Comment (andela-gike) [reply UI/js/all-meetup.js:4]: Okay

**Review by andela-gike — APPROVED** [review]

**Pull request merged**